api/utils/cfApiClient.js

```diff
@@ -339,6 +419,6 @@ CloudFoundryAPIClient.findEntity = findEntity;
 CloudFoundryAPIClient.firstEntity = firstEntity;
 CloudFoundryAPIClient.objToQueryParams = objToQueryParams;
 CloudFoundryAPIClient.buildRequestBody = buildRequestBody;
-CloudFoundryAPIClient.sleep = sleep;
+CloudFoundryAPIClient.sleep = wait;
```

Comment:
Do you want to rename throughout the app? I think VSCode might do it automatically

Reply:
We don't use it anywhere so I'll trash it.